test/ngResource/resourceSpec.js

```diff
@@ -338,6 +338,17 @@ describe("resource", function() {
   });
 
 
+  it('should handle url encoding in mapped params', function() {
```

Comment:
I'd prefer a more descriptive testcase name (e.g. `should encode & in query params unless in query param value` or something).